client/model.go

```diff
@@ -98,17 +98,17 @@ type ConfigurationVariableSchema struct {
 }
 
 type ConfigurationVariable struct {
-	ScopeId        string                      `json:"scopeId"`
-	Value          string                      `json:"value"`
-	OrganizationId string                      `json:"organizationId"`
-	UserId         string                      `json:"userId"`
-	IsSensitive    bool                        `json:"isSensitive"`
-	Scope          Scope                       `json:"scope"`
-	Id             string                      `json:"id"`
-	Name           string                      `json:"name"`
-	Description    string                      `json:"description"`
-	Type           ConfigurationVariableType   `json:"type"`
-	Schema         ConfigurationVariableSchema `json:"schema"`
+	ScopeId        string                       `json:"scopeId,omitempty"`
+	Value          string                       `json:"value"`
+	OrganizationId string                       `json:"organizationId,omitempty"`
+	UserId         string                       `json:"userId,omitempty"`
```

Comment:
idk what i feel about those
we need to add `userId` and `orgId` and if for some reason we didn't add those i think we want to get runtime error
no?

Reply:
they can be defined as required on the variable resource this is just the type for the client